popsycle/synthetic.py

```diff
@@ -3146,3 +3154,452 @@ def calc_f(lambda_eff):
     return f
 
 
+def angdist(ra1, dec1, ra2, dec2):
+    '''
+    Takes two spherical coordinates and determines the angluar distance between
+    them.
+    Input:
+    ra1,dec1 [degrees] angular coordinates of the first point
+    ra2,dec2 [degrees] angular coordinates of the second point
+    Output:
+    distance [degrees] angular distance between the two points
+    '''
+    from numpy import pi, sin, cos, arccos
+    d2r = pi / 180.0
+    cosdist = (cos(np.absolute(ra1 - ra2) * d2r) * cos(np.absolute(dec1 * d2r)) * cos(np.absolute(dec2 * d2r)) +
+                sin(np.absolute(dec1 * d2r)) * sin(np.absolute(dec2 * d2r)))
+    distance = arccos(cosdist) / d2r
+    return distance
+
+def rho_dmhalo(r, rho_0, r_s, gamma):
```

Comment:
I think it would be good to have the defaults in this line, since they are referred to else where. It is fine to also have them elsewhere (e.g. config file). Recommend changing this to:
`def rho_dmhalo(r, , rho_0=0.0106, r_s=19, gamma=1):`

Reply:
Didn't we decide that default gamma should equal .5?

Reply:
Or are we sticking with the regular NFW profile as default. (This was not good for the velocities though).

Reply:
Also going to add something similar I said above. I'm not sure if we should add these default values in. Just because we are using the McM17 paper doesn't mean that other people will want to use these values too? I'm mostly concerned because rho_0 and c (which determines what r_s is) change significantly between papers.

Reply:
Anyone who wants can go between c, r_s, r_vir, rho_0. The reason why McMillan (17) uses just r_s, and rho_0 is because that provides the most compact form of the density and thus is easiest to fit, I think that is a 4th reason to switch to them.

I agree that other people might not want to use the defaults, but I think that there will be a non-negligible group of people that appreciate that there are researched defaults, so that if they just want to run the code and get something reasonable they can do so easily (I know I would). Any super user that wants to investigate a given mode can easily overwrite these defaults through the config file.